changelog.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: Changelog
+description: A curated, chronologically ordered list of notable changes for each version of the project.
+nav_order: 99
+---
+
+# Changelog
+
+{% assign logs = site.changelog | reverse %}
+{% for log in logs %}
+{{ log }}
+{% endfor %}
```

Comment:
See note above about using markdown files in this way.

Also, either this file or `CHANGELOG.md` should be removed, as they duplicate one another and result in two entries in the menu.